### ISSUE TITLE
New plugin: Landmarks

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/ProjectVisBug#526]
Invokes [this script](https://github.com/matuzo/DevToolsSnippets/blob/main/landmarks/landmarks.js), or something like it. The outline style could be replaced with visbug handles and labels elements, see the [zindex plugin](https://github.com/GoogleChromeLabs/ProjectVisBug/blob/main/app/plugins/zindex.js#L23) for details 👍🏻 

I'd like to make this part of the accessibility inspect tool, but i'm not sure running it when the tool is picked is the right way. Any thoughts on how to make this easier to find?

**Comment by argyleink** [comment]
related #501

**Comment by hchiam** [comment]
Maybe have a button that appears on the bottom left, in the blank space under the `#border` color icon? 
<img src="https://user-images.githubusercontent.com/18131787/123189492-e4ca8980-d46b-11eb-83d9-9f74b4168c42.png" height="200">
The only thing is there's not much space underneath if your screen is small or if you're zoomed in, so maybe the button could go elsewhere on the screen? bottom right of screen? next to the Accessibility Tool `li`?

Or maybe have a toggle for landmarks as a keyboard shortcut and show that on the `<aside accessibility>`? It looks like `n` and `r` are available https://visbug.web.app/

**Comment by argyleink** [comment]
I like the idea of putting it into the accessibility tool, as a shortcut!
- [ ] update the a11y tooltip to show the new hotkey `ctrl`
- [ ] while a11y tool is selected, holding the control key shows the landmarks, releasing it hides them
- [ ] landmarks can be shown and hid
- [ ] landmarks use visbug handles or hover, and a visbug label

it places the action and ux right there with the accessibility inspect workflow. hover over stuff, verify contrast and attributes, then hold control and review landmarks for errors and proper flow. sounds awesome. do you think there should be a way to make the landmarks sticky, as in they persist beyond holding the hotkey? what's the use case where you need the landmarks to stick around?

**Comment by hchiam** [comment]
not a very strong reason, but: it could be helpful to remind you where the landmarks are while you tab and/or navigate around the page with the keyboard, if the landmark labels persist. or maybe only show the landmark that applies to the currently-focused element?

**Comment by argyleink** [comment]
nice, that sounds like a fun plugin or debug feature, like tab order breadcrumbs. these persistent marks could show a number too! but sounds like a different plugin? plus, tab/shift+tab are already used by visbug to traverse elements in the same way Sketch does, so we'd need to do some extra work in the plugin to steal tab.

**Comment by hchiam** [comment]
True, tab order breadcrumbs is prolly better off in a different plugin, if at all. In that case, only showing landmarks while you hold down `ctrl` makes sense to me now.
